definitions/ext-router/summary_metrics.yml

```diff
@@ -0,0 +1,41 @@
+ipAddress:
+  title: Router IP Address
+  unit: STRING
+  tag:
+    key: src_addr
+
+cpuUtilization:
+  title: CPU Utilization (%)
+  unit: PERCENTAGE
+  query:
+    select: average(kentik.snmp.CPU) AS 'CPU Utilization'
```

Comment:
The `as` here will not be really used anywhere. And I'm not sure how our summary metrics will handle that.
@gvassallo would appreciate your input on this one.

Reply:
i think this might break summary metrics or in general, it's not required

Reply:
@gvassallo | @naxhh - so I should remove all of the `'AS'` on these PRs for summary metrics only, or both summary and golden?

Reply:
I'd say so, we're gonna add the `as` to the golden metric queries by default in one of our next releases. I'd leave @naxhh making the final decision though :D

Reply:
@thezackm remove them all from both sides. We are in the process of automating this and will be easier for us.
Thanks for the patience!


Side note: I'll make a note to check SM behavior on this so we can add it to the linting.

Reply:
done and done. thanks!